.pre-commit-config.yaml

```diff
@@ -4,4 +4,18 @@ repos:
     rev: 2.1.1
     hooks:
     -   id: sqlfluff-lint
-    -   id: sqlfluff-fix
\ No newline at end of file
+    -   id: sqlfluff-fix
+  - repo: https://github.com/tconbeer/sqlfmt
+    rev: v0.19.2
+    hooks:
+      - id: sqlfmt
+        entry: >-
+          sqlfmt -v dbt
+            --exclude dbt/venv/**/*
+            --exclude dbt/models/**/*
+            --exclude dbt/target/**/*
+            --exclude dbt/dbt_packages/**/*
+            --exclude dbt/dbt_modules/**/*
+        files: ^dbt/
```

Comment:
The `files` attribute is required here, since otherwise passing the `--all-files` flag to `pre-commit` (as we do in CI) will run sqlfmt on _everything_ in this repo.

https://github.com/ccao-data/data-architecture/blob/5b8279a83a1bf7a127209311e34e3bc56373da82/.github/workflows/pre-commit.yaml#L36-L38

As an aside, I'm surprised that this is how pre-commit behaves, given that it claims [the `entry` command options will not be overriden](https://pre-commit.com/#hooks-entry). I'm guessing there's a special case for `file`, which must do some globbing of its own behind the scenes.